extensions/amp-bind/0.1/bind-validator.js

```diff
@@ -132,9 +133,9 @@ export class BindValidator {
     if (allowedProtocols && url) {
       const re = /^([^:\/?#.]+):[\s\S]*$/;
       const match = re.exec(url);
-
       if (match !== null) {
         const protocol = match[1].toLowerCase().trimLeft();
+        // hasOwnProperty() needed since nested objects are not prototype-less.
         if (!allowedProtocols.hasOwnProperty(protocol)) {
```

Comment:
It doesn't (so far):

![image](https://cloud.githubusercontent.com/assets/20525523/23968285/7db471e0-0998-11e7-9b2c-fd65431c0701.png)